next.config.js

```diff
@@ -1,8 +1,11 @@
 module.exports = {
+  experimental: {
+    outputStandalone: true,
+  },
   images: {
-    domains: process.env.NEXT_PUBLIC_IMAGE_DOMAINS.split(",")
-      .map((d) => d.trim())
-      .filter((d) => d),
+    domains: process.env.NEXT_PUBLIC_IMAGE_DOMAINS?.split(", ")
```

Comment:
Ola @kilemensi  is there something I'm overlooking here? Vercel complaints `SyntaxError: Unexpected token '.'` `domains: process.env.NEXT_PUBLIC_IMAGE_DOMAINS?.split(", ")`

Reply:
You shouldn't need anything extra @esirK. This is the same code as in TwoopsTrackerUI: https://github.com/CodeForAfrica/TwoopsTrackerUI/blob/b898b727481ec2da62e3dbfc4e93e8faf566fd09/next.config.js#L5-L9

And Next.js has supported in since [`v9`](https://nextjs.org/blog/next-9-1-7#support-for-the-latest-javascript-features)

Aside: There shouldn't be a space in `split`... just `","` and not `", "`.